rsciio/delmic/_api.py

```diff
@@ -1,80 +1,804 @@
+                        f"got {type(Scale)} instead."
+                    ) from e
+
+            if scale_value is not None and not (1e-15 < scale_value < 1):
+                raise ValueError(
+                    f"Scale value {scale_value} for axis {axis_name}"
+                    f"is outside of expected ranges."
+                )
+
+            # Handle the special cases for the C, T, and A axes
+            if axis_name == "C":
+                axis_dict = {
+                    "name": "Wavelength",
+                    "axis": Scale * unit_factor(prefix),  # Full axis loaded
+                    "units": prefix + "m",  # Adjusted unit prefix
+                    "navigate": False,  # No navigation for the C axis
+                }
+            elif axis_name == "T":
+                axis_dict = {
+                    "name": "Time",
+                    "axis": Scale * unit_factor(prefix),  # Full axis loaded
+                    "units": prefix + "s",  # Adjusted unit prefix
+                    "navigate": False,  # No navigation for the T axis
+                }
+            elif axis_name == "A":
+                ScaleA = np.arange(0, Image.shape[1])
+                axis_dict = {
+                    "name": "Angle",
+                    "axis": ScaleA,  # Full axis loaded, no scaling needed
+                    "units": "",  # Unit for Angle
+                    "navigate": False,  # No navigation for the A axis
+                }
+            else:
+                axis_dict = {
+                    "name": axis_name,
+                    "size": Image.shape[
+                        -(i + 1)
+                    ],
+                    "offset": float(np.array(ImgData.get(offset_key)))
+                    * unit_factor(prefix),
+                    "scale": scale_value * unit_factor(prefix),
+                    "units": prefix + "m",
+                    "navigate": True,
+                }
+            axes.append(axis_dict)
+
+    # Reorder axes to X, Y, Wavelength, Angle
+    axes_ordered = []
+    for axis_name in ["Y", "X", "Time", "Angle", "Wavelength"]:
+        for axis in axes:
+            if axis["name"] == axis_name:
+                axes_ordered.append(axis)
+                break
+
+    return axes_ordered
+
+
+def make_metadata(Acq):
+    """Create a metadata dictionary for a given acquisition (Acq)."""
+    metadata = {
+        "Signal": {
+            "quantity": "",  # Placeholder, to be filled with actual data
+            "signal_type": "",  # Placeholder, to be filled with actual data
+            "signal_origin": "",  # Placeholder, to be filled with actual data
+        }
+    }
+    if "quantity" in Acq:
+        metadata["Signal"]["quantity"] = Acq["quantity"]
+    else:
+        metadata["Signal"]["quantity"] = "Intensity (counts)"  # Default value
+
+    if "signal_type" in Acq:
+        metadata["Signal"]["signal_type"] = Acq["signal_type"]

```

Comment:
`signal_type` is used to select the right `signal class`. If LumiSpy is installed, it should default to the correct signal classes by choosing `Luminescence` for anything where the only signal axis is wavelength (spectra or hyperspectral maps), `LumiTransient` if the only signal axis is time and `LumiTransientSpectrum` for streak camera images.

See e.g. 
https://github.com/hyperspy/rosettasciio/blob/1274ed5bc4a8e9b73b5fcc241d7d989429ffee6f/rsciio/jobinyvon/_api.py#L85
https://github.com/hyperspy/rosettasciio/blob/332412f93b27c014311a84b13eea9852dfb2707f/rsciio/hamamatsu/_api.py#L362